Examples/Io/Root/include/ACTFW/Io/Root/RootMaterialTrackWriter.hpp

```diff
@@ -55,6 +55,8 @@ class RootMaterialTrackWriter
     bool prePostStep = false;
     /// Write the surface to which the material step correpond
     bool storesurface = false;
```

Comment:
Again, here CamelCase would be better, i.e. `storeSurface`

Reply:
Done